test/functional/wallet_basic.py

```diff
@@ -283,7 +283,7 @@ def run_test(self):
         sync_blocks(self.nodes[0:3])
         node_2_bal += 2
 
-        #tx should be added to balance because after restarting the nodes tx should be broadcastet
+        #tx should be added to balance because after restarting the nodes tx should be broadcasted
```

Comment:
nit: personal preference for broadcast as the past participle of broadcast (both are acceptable but broadcast appears to be more common: https://english.stackexchange.com/questions/33207/broadcast-or-broadcasted)